AutWebTest/AutWebTest/Tests/LoginPageTests.cs

```diff
@@ -0,0 +1,75 @@
+using AutWebTest.Pages;
+using AutWebTest.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace AutWebTest.Tests
+{
+    public class LoginPageTests
+    {
+        private IWebDriver driver;
+        private LoginPage loginPage;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            driver.Url = "http://localhost:5000/";
+            loginPage = new LoginPage(driver);
+        }
+
+        [TearDown]
+        public void TearDown() => driver.Dispose();
+
+        [Test]
+        public void LoginWithEmptyField()
+        {
+            loginPage.Login(string.Empty, string.Empty);
+            Assert.AreEqual("User not found!", loginPage.ErrorMessage);
+        }
```

Comment:
You have lots of similar tests. You can combine them like this:
```suggestion
        [TestCase(string.Empty, string.Empty, "User not found!")]
        [TestCase(string.Empty, "newyork1", "User not found!")]
        [TestCase("test", string.Empty, "User not found!")]
        public void NegativeLogin(string login, string password, string expectedError)
        {
            loginPage.Login(login, password);
            Assert.AreEqual(expectedError, loginPage.ErrorMessage);
        }
```

Reply:
Good suggestion, thank you very much!